src/share/routes.js

```diff
@@ -128,18 +132,42 @@ function register(router) {
         }
 
         const {header, content, isEmpty} = contentRenderer.getContent(note);
-
         const subRoot = getSharedSubTreeRoot(note);
+        const opts = {note, header, content, isEmpty, subRoot, assetPath, appPath};
```

Comment:
This is all currently passed to the user's custom template as well, should we only pass a subset to the user? Or are there other things that would be useful to the user template that are not currently passed?

Either way this creates a sort of pseudo-api so it would be good to be certain of what should be passed and try not to change it much going forward. But I would still call user templates an "at your own risk" type of extension.